### PR TITLE
chore: initial empty scaffold project

### PR DESCRIPTION
**Pull request by benoitf**
init step for https://github.com/containers/podman-desktop/issues/8972

you'll get an empty screen

Change-Id: Ie39624fa2bb98774a44cdfab0897327cd7b1b63e

### CODE REVIEW

**Comment by benoitf** [comment]
ready for review, checks are ✅

File: eslint.config.mjs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    },
+    settings: {
+      'import/resolver': {
+        typescript: true,
+        node: true,
+
+        'eslint-import-resolver-custom-alias': {
+          alias: {
+            '/@': './src',
+            '/@gen': './src-generated',
```
Comment (axel7083) [inline eslint.config.mjs:80]: `src-generated` ? we do not have generated stuff here ?

File: src/podman-desktop.css [hunk 2]
```diff
@@ -0,0 +1,483 @@
+@scope (.light) {
```
Comment (axel7083) [inline src/podman-desktop.css:1]: not a blocker, jsut a remark: I am wondering if we should not publish this somewhere, as we will always need to update it to keep it updated :(

**Review by axel7083 — APPROVED** [review]
LGTM 🚀

File: src/podman-desktop.css (hunk 2 above)
Comment (benoitf) [reply src/podman-desktop.css:1]: yes, it's my intent 👍

File: eslint.config.mjs (hunk 1 above)
Comment (benoitf) [reply eslint.config.mjs]: removed

**Review by deboer-tim — APPROVED** [review]
LGTM

**Review by cdrage — APPROVED** [review]
LGTM here too for first interation, let's test this out

**Pull request merged**